staging/src/k8s.io/apiserver/pkg/registry/generic/rest/streamer.go

```diff
@@ -61,11 +62,15 @@ func (s *LocationStreamer) InputStream(ctx context.Context, apiVersion, acceptHe
 	if transport == nil {
 		transport = http.DefaultTransport
 	}
+
 	client := &http.Client{
 		Transport:     transport,
 		CheckRedirect: s.RedirectChecker,
 	}
 	req, err := http.NewRequest("GET", s.Location.String(), nil)
+	if err != nil {
```

Comment:
Split this into a separate PR?  I'm having bad flashbacks.

Reply:
Good point. Just hate uncommented errors which are ignored and likely to cause a problem. I agree it should be a separate PR.